CombinatorialGames/Game/Special.lean

```diff
@@ -130,4 +130,27 @@ theorem star_fuzzy_down : star ‖ down := by
   rw [← neg_fuzzy_neg_iff, neg_down, neg_star]
   exact star_fuzzy_up
 
+section TinyMiny
+
+variable (x : PGame)
+
+/-! ## Tiny and Miny -/
+
+/-- A tiny game ⧾ is defined as {0 | {0 | -G}}, and is amongst the smallest of the infinitesimals. -/
+def tiny : PGame :=
+  ⟨PUnit, PUnit, fun _ ↦ 0, fun _ ↦ ⟨PUnit, PUnit, fun _ ↦ 0, fun _ ↦ -x⟩⟩
+
+/-- A miny game ⧿ is defined as {{G | 0} | 0}. -/
+def miny : PGame :=
+  ⟨PUnit, PUnit, fun _ ↦ ⟨PUnit, PUnit, fun _ ↦ x, fun _ ↦ 0⟩, fun _ ↦ 0⟩
+
+theorem miny_eq_neg_tiny : miny x = -tiny x := by
+  rw [miny, tiny]
+  simp
+
+/-- **Tiny is tiny**. The tiny games are among the smallest of the infinitesimals. -/
+proof_wanted gt_tiny (x : PGame) (hx : 0 < x) : ∃ n : ℕ, tiny n < x
```

Comment:
Maybe?
```suggestion
proof_wanted exists_tiny_lt_of_pos (x : PGame) (hx : 0 < x) : ∃ n : ℕ, tiny n < x
```

Reply:
Note that `n : PGame` is *not* the same or even identical to `Ordinal.toPGame n`, though it is equivalent. Does that change anything?